framework/source/class/qx/bom/Cookie.js

```diff
@@ -62,7 +62,13 @@ qx.Bootstrap.define("qx.bom.Cookie",
         end = document.cookie.length;
       }
 
-      return unescape(document.cookie.substring(len, end));
+      try {
+        return decodeURIComponent(document.cookie.substring(len, end));
+      }
+      catch (URIError) {
+        console.error(URIError.message);
```

Comment:
That should be something like:

`qx.log.Logger.error("Error decoding URI components", URIError.message);`